.circleci/unittest/windows/scripts/set_cuda_envs.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+if [ "${CU_VERSION:-}" != "cpu" ] ; then
+    if [[ ${#CU_VERSION} -eq 4 ]]; then
+        CUDA_VERSION="${CU_VERSION:2:1}.${CU_VERSION:3:1}"
+    elif [[ ${#CU_VERSION} -eq 5 ]]; then
+        CUDA_VERSION="${CU_VERSION:2:2}.${CU_VERSION:4:1}"
+    fi
```

Comment:
I don't think `CU_VERSION` is defined anywhere, so could you please explain why this block is needed

Reply:
1. the cuda version is needed to set cuda environment varilables, such as CUDA_PATH.
2. I add logs to check nvcc version and path, that needs cuda installed .
I think it should be called where CU_VERSION is defined.

Reply:
CU_VERSION and CUDA_VERSION confused me.
I make a little change to make the CI work in #4145 
Maybe we need to make it clearer in the future.